### PR TITLE
IDEMPIERE-6306 : Improve Calendar UI

### PR DESCRIPTION
**Pull request by uthadehikaru**
https://idempiere.atlassian.net/browse/IDEMPIERE-6306

# Pull Request Checklist

- [x] My code follows the [code guidelines](https://wiki.idempiere.org/en/Contributing_to_Trunk) of this project
- [x] My code follows the best practices of this project
- [x] I have performed a self-review of my own code
- [x] My code is easy to understand and review. 
- [x] I have checked my code and corrected any misspellings
- [x] In hard-to-understand areas, I have commented my code.
- [x] My changes generate no new warnings
### Tests
- [x] I have tested the direct scenario that my code is solving
- [x] I checked all collaterals that can be affected by my changes, and tested other potential affected scenarios
- [ ] New and existing unit tests pass locally with my changes
- [ ] I have added unit tests that prove my fix is effective or that my feature works
### Documentation
- [ ] I have made corresponding changes to the documentation as follows:
- - [ ] New feature (non-breaking change which adds functionality): I have created the New Feature page in the project wiki explaining the functionality and how to use it. If relevant, I have committed sample data to the core seed to have usable examples in GardenWorld.
- - [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected): I have documented the change in a clear way that everyone in the community can understand the impact of the change.
- - [ ] Improvement (improves and existing functionality): This documentation is needed if the improvement changes the way the user interacts with the system or the outcome of a process/task changes. If it is just, for instance, a performance improvement, documentation might not be needed. 
- [ ] The changed/added documentation is in the project wiki (not privately-hosted pdf files or links pointing to a company website) and is complete and self-explanatory.

### CODE REVIEW

File: org.idempiere.zk.iceblue_c.theme/src/web/theme/iceblue_c/zul/calendar/calendar.css [hunk 1]
```diff
@@ -691,26 +691,31 @@
...
-    border-top: 4px solid #FFFFFF;
-    color: #000000;
-    line-height: 16px;
-    overflow: hidden;
-    padding-right: 2px;
-    text-align: center;
-    font-size: 14px;
+	background-color: transparent;
+	/* border-left: 4px solid #FFFFFF; */
+	/* border-top: 4px solid #FFFFFF; */
```
Comment (hengsin) [inline org.idempiere.zk.iceblue_c.theme/src/web/theme/iceblue_c/zul/calendar/calendar.css:712]: remove, don't remark.

**Review by hengsin — CHANGES_REQUESTED** [review]
as usual, after making changes to css, you need to update lang-addon.xml

**Comment by hengsin** [comment]
hi @uthadehikaru , the following patch make clicking on the day number trigger create ticket event:
[dpcalendar.patch.txt](https://github.com/user-attachments/files/17841093/dpcalendar.patch.txt)

please merge it if tested ok.

**Comment by uthadehikaru** [comment]
> hi @uthadehikaru , the following patch make clicking on the day number trigger create ticket event: [dpcalendar.patch.txt](https://github.com/user-attachments/files/17841093/dpcalendar.patch.txt)
> 
> please merge it if tested ok.

tested ok and I also implement on calendar window

**Review by hengsin — APPROVED** [review]
tested

**Pull request merged**